### PR TITLE
Update generated code in existing testcases

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

### CODE REVIEW

File: grpc-ballerina/tests/05_invalid_resource_client.bal [hunk 1]
```diff
@@ -43,79 +43,4 @@ function testInvalidInputParameter() {
...
-            message = req;
-        }
-        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld98/testInt", message, headers);
-        [anydata, map<string|string[]>][result, respHeaders] = payload;
-        
-        return {content: <int>result, headers: respHeaders};
-    }
-
-}
+}
```
Comment (BuddhiWathsala) [inline grpc-ballerina/tests/05_invalid_resource_client.bal:46]:
```suggestion
}

```

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Comment by BuddhiWathsala** [comment]
There are plenty of commented lines in tests. Do we keep it to easily run tests manually?. And indicate the user that these kinds of generated code is there?.

**Comment by daneshk** [comment]
> There are plenty of commented lines in tests. Do we keep it to easily run tests manually?. And indicate the user that these kinds of generated code is there?.

yes, we kept it to make manual testing easier. in `grpc_tooling_common_test.bal` file, there is a commented test case. we need to uncomment and run the testcase. this will replace all generated files in the test case. we can check the diff easily. there are duplicated types in tests, we commented them to make it easy to compare the diff.

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**